Makefile.am

```diff
@@ -18,6 +18,21 @@ service_in_files = \
 	data/com.endlessm.DiscoveryFeed.service.in \
         $(NULL)
 
+gsettings_SCHEMAS = data/com.endlessm.DiscoveryFeed.gschema.xml
+
+%.gschema.xml: %.gschema.xml.in Makefile
+	$(AM_V_GEN) sed -e 's|@GETTEXT_PACKAGE[@]|$(GETTEXT_PACKAGE)|g' \
+	$< > $@ || rm $@
+
+@GSETTINGS_RULES@
+
+# We need to compile schemas at make time
+# to run from source tree
+gschemas.compiled: $(gsettings_SCHEMAS:.xml=.valid)
+	$(AM_V_GEN) $(GLIB_COMPILE_SCHEMAS) --targetdir=data data
```

Comment:
I don't think this part is required; we do not support running the app completely uninstalled anyway.